src/mmw/apps/modeling/views.py

```diff
@@ -494,3 +494,58 @@ def boundary_layer_detail(request, table_code, obj_id):
             return Response(geojson)
         else:
             return Response(status=status.HTTP_400_BAD_REQUEST)
+
+
+@decorators.api_view(['GET'])
+@decorators.permission_classes((AllowAny, ))
+def plot_point_source(request):
```

Comment:
This dataset is updated rarely, let's set aggressive cache headers on this endpoint.